packages/p2p/src/components/orders/order-details/order-details-footer.jsx

```diff
@@ -149,7 +149,7 @@ const OrderDetailsFooter = ({ order_information, cancelPopup, showPopup }) => {
                     </div>
                     <div className='order-details__footer--right'>
                         <Button large primary onClick={showReceivedFundsPopup}>
-                            {localize("I've received funds")}
+                            {localize("I've received payment")}
```

Comment:
Just checking, have you confirmed this with CW team?

Reply:
It's coming from the design.

Reply:
This doesn't sound right to me, can you please confirm with them? `I've received the payment` would be grammatically correct.

Reply:
It sounds correct to me, but let me confirm with them.